mockCRM.js

```diff
@@ -0,0 +1,67 @@
+// This function returns mock CRM data and is being used as an analog for your functionality being added to the plugin.
```

Comment:
Add a line here saying something like "this file provides fake data for display purposes. You should replace this with your own actual data relevant to your application"